build.gradle.kts

```diff
@@ -149,4 +161,20 @@ if (signingPropsFile.exists()) {
             project.ext.set(key, value)
         }
     }
-}
\ No newline at end of file
+}
+
+android {
+    namespace = "ca.gosyer"
+
+    sourceSets["main"].manifest.srcFile("src/androidMain/AndroidManifest.xml")
+
+    defaultConfig {
+        compileSdk = 34
+        minSdk = 24
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_17
+        targetCompatibility = JavaVersion.VERSION_17
```

Comment:
I would prefer to target Java 8 or 11, don't want to cut a bunch of users out.

Reply:
I have modified the target to 1.8 and tested the imported dependencies in the gradle6.1.1-agp4.0.2-jdk1.8 project and compiled successfully.